src/main/java/frc/robot/subsystems/AprilTag.java

```diff
@@ -95,6 +96,32 @@ public record VisionParameters(
   public static final RobotPreferences.BooleanValue ENABLE_TAB =
       new RobotPreferences.BooleanValue("AprilTag", "Enable Tab", true);
 
+  private enum PoseEstimationStrategy {
+    AverageBestTargets(PoseStrategy.AVERAGE_BEST_TARGETS),
+    ClosestToCameraHeight(PoseStrategy.CLOSEST_TO_CAMERA_HEIGHT),
+    ClosestToLastPose(PoseStrategy.CLOSEST_TO_LAST_POSE),
+    ClosestToReferencePose(PoseStrategy.CLOSEST_TO_REFERENCE_POSE),
+    ConstrainedSolvePnp(PoseStrategy.CONSTRAINED_SOLVEPNP),
+    LowestAmbiguity(PoseStrategy.LOWEST_AMBIGUITY),
+    MultiTagPnpOnCoprocessor(PoseStrategy.MULTI_TAG_PNP_ON_COPROCESSOR),
+    PnpDistanceTrigSolve(PoseStrategy.PNP_DISTANCE_TRIG_SOLVE);
+
+    private final PoseStrategy strategy;
+
+    private PoseEstimationStrategy(PoseStrategy strategy) {
+      this.strategy = strategy;
+    }
+
+    public PoseStrategy getStrategy() {
+      return strategy;
+    }
+  }
+
+  @RobotPreferencesValue(column = 0, row = 1)
+  public static EnumValue<PoseEstimationStrategy> POSE_ESTIMATION_STRATEGY =
+      new EnumValue<PoseEstimationStrategy>(
+          "AprilTag", "Pose Est. Strategy", PoseEstimationStrategy.AverageBestTargets);
```

Comment:
Right now, we're set up to use `MultiTagPnpOnCoprocessor`. Should we make that the default?